src/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.jsx

```diff
@@ -199,23 +225,36 @@ const DiffExprCompute = (props) => {
     <Form size='small' layout='vertical'>
 
       <Radio.Group onChange={(e) => {
-        setType(e.target.value);
 
-        setSelectedGroups({
-          cellSet: null,
-          compareWith: null,
-          basis: null,
-        });
-      }} value={type}>
-        <Radio style={radioStyle} value={ComparisonType.between} disabled={!hasMetadata}>
+        dispatch(setComparisonType(e.target.value));
+
+        // If comparisonType key is not available, add to group
+        if(!comparisonGroup[e.target.value]) {
+          dispatch(setComparisonGroup({
+            type: e.target.value,
+            cellSet: null,
+            compareWith: null,
+            basis: null,
+            ...comparisonGroup[e.target.value]
+          }))
+        }
```

Comment:
I think we can go without dispatching this one -- what is the point of dispatching an action to save empty cell set selections?

Reply:
It's actually to initialize the `between` group. Actually it can be set as an initial value. However I was thinking for the case if we want to add another way to compare. But that may doesn't seem to be the case. I'll move it as an initial value.